### PR TITLE
Updated documentation of Exchange log collector to include  LogStartDate  and LogEndDate 

### PR DESCRIPTION
**Pull request by iserrano76**
**Issue:**
Updated documentation of Exchange log collector to include  LogStartDate  and LogEndDate added in #1333

### CODE REVIEW

File: docs/Diagnostics/ExchangeLogCollector.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ GetVdirs | Enable to collect the Virtual Directories of the environment.
 HighAvailabilityLogs | Enable to collect High Availability Logs. Windows Event Logs like: `Microsoft-Exchange-HighAvailability`, `Microsoft-Exchange-MailboxDatabaseFailureItems`, and `Microsoft-Windows-FailoverClustering`
 HubConnectivityLogs | Enable to collect the Hub connectivity logging. Location: `(Get-TransportService $server).ConnectivityLogPath`
 HubProtocolLogs | Enable to collect the protocol logging. Location: `(Get-TransportService $server).ReceiveProtocolLogPath` and `(Get-TransportService $server).SendProtocolLogPath`
-IISLogs | Enable to collect IIS Logs and HTTPErr Logs from the Exchange Server. Default Location: `C:\inetpub\logs\LogFiles\W3SVC1`, `C:\inetpub\logs\LogFiles\W3SVC1`, and `C:\Windows\System32\LogFiles\HTTPERR`. Only able to collect on `LogAge`.
+IISLogs | Enable to collect IIS Logs and HTTPErr Logs from the Exchange Server. Default Location: `C:\inetpub\logs\LogFiles\W3SVC1`, `C:\inetpub\logs\LogFiles\W3SVC2`, and `C:\Windows\System32\LogFiles\HTTPERR`. Not able to collect on DaysWorth and HoursWorth
```
Comment (dpaulson45) [inline docs/Diagnostics/ExchangeLogCollector.md:88]: Why was this changed to `not` able to collect? Was something changed where IIS Logs are able to collect the entire directory?

Comment (iserrano76) [reply docs/Diagnostics/ExchangeLogCollector.md:88]: Hi David, 
As the previous doc, shows “Only able to collect on `LogAge`.” I guessed that it does not work with other options, but you are right. 
I was checking the script with all options ( LogAge, DaysWorth, HoursWorth and LogStart/EndDate ) and works correctly with IIS logs. 
I will remove the restriction comment.

Comment (iserrano76) [reply docs/Diagnostics/ExchangeLogCollector.md]: Oh, ok. Now I understood it. 
In the original doc we have Only able to collect on `LogAge`. And I link that information with the Log Age parameter, not with the age of every log.
I can leave as it was before but maybe something like this could be more clarifier:
IISlogs do not collect all logs on CollectAllLogsBasedOnLogAge:$false, just works based on log age.
Let me know your thoughts and I will set as you prefer.

**Review by dpaulson45 — APPROVED** [review]
